src/pages/dev/send-tokens/interchain-tokens/upgrade-tokens.mdx

```diff
@@ -4,115 +4,33 @@ import { Callout } from "/src/components/callout";
-
-If pre-mint is needed, you must make ERC20 approval to the Interchain Token Factory contract. When tokens are moved from the origin chain to another chain, the token will be locked on the origin chain and minted on the destination chain. If you moved tokens directly from one non-origin chain to another, the token would be burned on the source chain and minted on the destination chain.
+1. Create and register an ERC-20 token on the [Interchain Token Portal](https://testnet.interchain.axelar.dev/).
+1. [Programmatically create a canonical token](/dev/send-tokens/interchain-tokens/developer-guides/programmatically-create-a-canonical-token).
 
 ## Link deployed tokens on multiple chains into Interchain Token(s)
 
-For custom functionality on multiple chains:
-
-- Deploy your [custom token](https://docs.axelar.dev/dev/glossary#custom-token) on multiple chains or have existing versions on multiple chains
-- Deploy a [Mint/Burn](https://docs.axelar.dev/dev/glossary#mintburn) - [Token Manager](https://docs.axelar.dev/dev/glossary#token-manager) for existing tokens on all chains using [`deployTokenManager`](https://github.com/axelarnetwork/interchain-token-service/blob/9edc4318ac1c17231e65886eea72c0f55469d7e5/contracts/InterchainTokenService.sol#L276) on Interchain Token Service. The `deployTokenManager` method requires parameters like `salt`, `destinationChain`, [`tokenManagerType`](https://github.com/axelarnetwork/interchain-token-service/blob/main/contracts/interfaces/ITokenManagerType.sol), `params`, and `gasValue`
-- Transfer Interchain Tokens between chains via the Interchain Token Service by calling the [`interchainTransfer`](https://github.com/axelarnetwork/interchain-token-service/blob/9223108211b977d9138fdd67d5b4a641fc35c18c/contracts/interfaces/IInterchainTokenService.sol#L210)
-
-You can optionally have any of these custom tokens extend [`IInterchainTokenStandard`](https://github.com/axelarnetwork/interchain-token-service/blob/main/contracts/interfaces/IInterchainTokenStandard.sol) to offer [`interchainTransfer`](https://github.com/axelarnetwork/interchain-token-service/blob/a2dfcb2490497e627b66be789d944ec3260c5eea/contracts/interfaces/IInterchainTokenStandard.sol#L20) and [`interchainTransferFrom`](https://github.com/axelarnetwork/interchain-token-service/blob/a2dfcb2490497e627b66be789d944ec3260c5eea/contracts/interfaces/IInterchainTokenStandard.sol#L37) methods directly on your token.
-
-> You could decide to make all of these Token Managers [Mint/Burn](https://docs.axelar.dev/dev/glossary#mintburn), or you could specify at most one (likely your primary/origin chain) to be [Lock/Release](https://docs.axelar.dev/dev/glossary#lockunlock) as indicated in the [`ITokenManagerType.sol`](https://github.com/axelarnetwork/interchain-token-service/blob/main/contracts/interfaces/ITokenManagerType.sol) depending on your needs.
```

Comment:
This info is also very important. I see you have merged the info to talk about making the primary chain the origin chain, but referencing the token manager type is missing.